src/index.js

```diff
@@ -1,11 +1,63 @@
 #!/usr/bin/env node
 
-require('yargs')
+/**
+ * Networks available
+ */
+let networks = {
+  'm': 'https://mainnet.infura.io',
+  'ro': 'https://ropsten.infura.io',
+  'ri': 'https://rinkeby.infura.io',
+  'k': 'https://kovan.infura.io',
+  's': 'https://sokol.poa.network',
+  'p': 'https://core.poa.network'
+}
+
+let yargs = require('yargs')
   .option('url', {
     description: 'URL of the ethereum node to connect',
     default: 'http://localhost:8545'
   })
-  .command('completion', 'Generate bash completion script', yargs => {
+  .option('mainnet', {
+    alias: 'm',
```

Comment:
I'd prefer not having aliases for this. Just the full `--mainnet` flag. They are easier to remember than the aliases and, besides, options like `-m` or `-k` could be useful for other options in the future.

Reply:
I remove the aliases and leave the flags, added local option